pages/index.tsx

```diff
@@ -85,7 +74,7 @@ export default function Home() {
 				<div className="my-16 w-full flex flex-col items-center">
 					<div className="max-w-4xl w-full mx-4">
 						<div>
-							<TextHeader text="Featured projects" />
+							<TextHeader text="Ongoing projects" />
```

Comment:
Featured is not supposed to show all ongoing, it's supposed to show three selected by the admin. This hasn't been added yet, so it used the first three in the db. But, we can leave it like this I guess.